pkg/lvm/lvm_util.go

```diff
@@ -571,24 +690,87 @@ func getLvDeviceName(path string) (string, error) {
 //It returns LogicalVolume.
 //
 //Example: LogicalVolume{
-//		Name:     "pvc-213ca1e6-e271-4ec8-875c-c7def3a4908d",
-//		Size:     3221225472,
-//		Path:     "/dev/linuxlvmvg/pvc-213ca1e6-e271-4ec8-875c-c7def3a4908d",
+//		Name:               "pvc-082c7975-9af2-4a50-9d24-762612b35f94",
+//		FullName:           "vg_thin/pvc-082c7975-9af2-4a50-9d24-762612b35f94"
+//		UUID:               "FBqcEe-Ln72-SmWO-fR4j-t4Ga-1Y90-0vieKW"
+//		Size:                4294967296,
+//		Path:                "/dev/vg_thin/pvc-082c7975-9af2-4a50-9d24-762612b35f94",
+//		DMPath:              "/dev/mapper/vg_thin-pvc--082c7975--9af2--4a50--9d24--762612b35f94"
+//		Device:              "dm-5"
+//		VGName:              "vg_thin"
+//		SegType:             "thin"
+//		Permission:          1
+//		BehaviourWhenFull:   -1
+//		HealthStatus:        0
+//		RaidSyncAction:      -1
+//		ActiveStatus:        "active"
+//		Host:                "node1-virtual-machine"
+//		PoolName:            "vg_thin_thinpool"
+//		UsedSizePercent:     0
+//		MetadataSize:        0
+//		MetadataUsedPercent: 0
+//		SnapshotUsedPercent: 0
 //	}
 func parseLogicalVolume(m map[string]string) (LogicalVolume, error) {
 	var lv LogicalVolume
 	var err error
+	var sizeBytes int64
+	var count float64
 
 	lv.Name = m["lv_name"]
+	lv.FullName = m["lv_full_name"]
```

Comment:
better to declare them as constants.

Reply:
Only "lv_full_name" or every lv field? Why do we need constant for this only? For everything else it's is hard-coded

Reply:
we should use contants for everything.